recipes/monetdblite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "monetdblite" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hannesmuehleisen/MonetDBLite-Python/archive/v{{ version }}.tar.gz
+  sha256: 1e091afdb45a892f9790bf858285d7fcd32e00ccbfcba9b454ed803ceeca6c16
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
```

Comment:
Following the [README](https://github.com/hannesmuehleisen/MonetDBLite-Python), let me know if I should use `pip` anyway.

Reply:
We always try to use pip whenever possible b/c pip is the "future" of python packaging. You can try later on the feedstock.